test/EventHandler/main.cpp

```diff
@@ -358,6 +364,7 @@ void runTest(void)
 
     printf ("Final MyArg instance count (should be 0): %d\r\n", MyArg::instcount);
     printf ("\r\nTest Complete\r\n");
+    MBED_HOSTTEST_RESULT(true);
```

Comment:
I'd make this `MBED_HOSTTEST_RESULT(MyArg::instcount==0)`, to give it at least a bit of meaning :)